src/topics/using-containers.md

```diff
@@ -52,8 +52,10 @@ used a container called `node:slim`.
 Provide a "hello.js" and invoke `cwltool` providing the tool description and the
 input object on the command line:
 
-```{code-block} console
-$ echo "console.log(\"Hello World\");" > hello.js
+```{literalinclude} /_include/cwl/using-containers/hello.js
```

Comment:
Hmm.. the include doesn't seem to be working: https://common-workflow-languageuser-guide--306.org.readthedocs.build/en/306/topics/using-containers.html